docs/source/tutorial_basis_generation.md

```diff
@@ -0,0 +1,811 @@
+for which the best-approximation error is quite large.
+
+So far we completely neglected that the snapshot vectors may lie in a Hilbert space
+with some given inner product. To account for that, instead of the snapshot matrix
+{math}`A`, we consider the linear mapping that sends the {math}`i`-th canonical basis vector
+{math}`e_k` of {math}`\mathbb{R}^K` to the vector {math}`u(\mu_k)` in the
+{attr}`~pymor.models.interface.Model.solution_space`:
+
+$$ \Phi: \mathbb{R}^K \to V, \ e_k \mapsto u(\mu_k).$$
+
+Also for this finite-rank (hence compact) operator there exists a SVD of the form
+
+$$ \Phi(v) = \sum_{i=1}^r u_i \cdot \sigma_i \cdot (v_i, v) \qquad \forall v \in \mathbb{R}^K,$$
+
+with orthonormal vectors {math}`u_i` and {math}`v_i` that generalizes the SVD of a matrix.
+
+The POD in this more general form is implemented in pyMOR by the
+{meth}`~pymor.algorithms.pod.pod` method, which can be called as follows:
+
+```{code-cell}
+pod_basis, pod_singular_values = pod(U, product=fom.h1_0_semi_product, modes=25)
+```
+
+We said that the POD modes (left-singular vectors) are orthonormal with respect to the
+inner product on the target Hilbert-space. Let's check that:
+
+```{code-cell}
+np.linalg.cond(pod_basis.gramian(fom.h1_0_semi_product))
+```
+
+Now, let us compare how the POD performs against the greedy algorithm in the worst-case
+best-approximation error:
+
+```{code-cell}
+pod_errors = compute_proj_errors_orth_basis(pod_basis, V, fom.h1_0_semi_product)
+
+plt.figure()
+plt.semilogy(trivial_errors, label='trivial')
+plt.semilogy(greedy_errors, label='greedy')
+plt.semilogy(pod_errors, label='POD')
+plt.ylim(1e-1, 1e1)
+plt.legend()
+plt.show()
+```
+
+As it turns out, the POD spaces perform even slightly better than the greedy spaces.
+Why is that the case? Note that for finite training or validation sets, both considered error measures
+are equivalent. In particular:
+
+```{math}
+\sup_{k = 1,\ldots,K} \inf_{v \in V_N} \|u(\mu_k) - v\| \leq
+\left[\sum_{k = 1}^K \inf_{v \in V_N} \|u(\mu_k) - v\|^2\right]^{1/2} \leq
+\sqrt{K} \cdot \sup_{k = 1,\ldots,K} \inf_{v \in V_N} \|u(\mu_k) - v\|.
+```
+
+Since POD spaces are optimal in the {math}`\ell^2`-error, they miss the error of the
+optimal space in the Kolmogorov sense by at most a factor of {math}`\sqrt{K}`, which in
+our case is {math}`5`. On the other hand, the greedy algorithm also only produces
+quasi-optimal spaces. -- For very large training sets with more complex parameter dependence,
+differences between both spaces may be more significant.
+
+Finally, it is often insightful to look at the POD modes themselves:
+
+```{code-cell}
+fom.visualize(pod_basis)
+```
+
+As you can see, the first (more important) basis vectors account for the approximation of
+the solutions in the bulk of the subdomains, whereas the higher modes are responsible for
+approximating the solutions at the subdomain interfaces.
+
+
+.. _weakgreedy:
```

Comment:
```suggestion
```

Reply:
IIRC this produces a (used) link anchor and shouldn't be removed.

Reply:
Oh yes, you are right. But is it possible to hide that in the output?

Reply:
Ah, didn't see it in the output before. Maybe it can be removed after all, since the headline does produce its own anchor (now?) https://docs.pymor.org/docs-myst/tutorial_basis_generation.html#weak-greedy-algorithm
There doesn't seem to be a reference to `weakgreedy` anywhere else. I'll remove them.

Reply:
https://github.com/pymor/pymor/pull/1323/commits/3c33554261de7a8bdbf200ca5ad20ebb54f38fa5

Reply:
Would be great if that works!